### PR TITLE
Fix shipping modul update

### PR DESCRIPTION
**Pull request by grandeljay**
Currently it is not possible to update shipping modules. This PR fixes that.

### CODE REVIEW

File: new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/StdModule.php [hunk 1]
```diff
@@ -430,8 +430,14 @@ private function invokeAction()
     private function renderButton($functionName, $buttonName)
     {
         $url = xtc_href_link(
-            FILENAME_MODULE_EXPORT,
-            'set=system&module=' . $this->code . '&moduleaction=' . $functionName
+            pathinfo($_SERVER['SCRIPT_NAME'], PATHINFO_BASENAME),
```
Comment (RobinTheHood) [inline new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/StdModule.php:433]: Should we test if $_SERVRE['SCRIPT_NAME'] is set?
```php
$scriptName = $_SERVRE['SCRIPT_NAME'] ?? ...
```

File: new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/StdModule.php [hunk 2]
```diff
@@ -430,8 +430,14 @@ private function invokeAction()
...
+            http_build_query(
+                [
+                    'set' => $_GET['set'],
```
Comment (RobinTheHood) [inline new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/StdModule.php:436]: Same here ... should we test if $_GET['set'] is set?
```php
$getSet = $_GET['set'] ?? ...
```

**Review by RobinTheHood — COMMENTED** [review]
Overall a good PR. Maybe you can still edit the marked positions. :-)

**Comment by grandeljay** [comment]
I don't currently see an advantage of using `$_GET['set'] ?? ''`. It would prevent a warning but still lead to unexpected/buggy behaviour.

I have made another suggestion with a fallback option in case the globals are not set.

As always, I am looking forward to your feedback!

**Pull request merged**